### DISCUSSION TITLE
`include_bytes!()` from outside project directory

### DISCUSSION THREAD

**Discussion by NotQuiteApex** [cross-rs/cross#1586] [Q&A] [answered]
Howdy! I'm wanting to use cross to compile my egui application for Windows from my Linux machine. Currently, I use `include_bytes!()` to add the app logo (i.e. `&include_bytes!("../../assets/applogo.png")[..]`, but I keep it outside of the project directory for use in other elements in my repo. Is it possible to include that directory for compilation in some way?

**Comment by Emilgardis** [comment, marked as answer]
Yes, use [`build.env.volume`](https://github.com/cross-rs/cross/blob/main/docs/config_file.md#buildenv)

So either something like

```
[build.env]
volume = ["ASSETS=path/to/assets"]
```

or skip setting it with `=` and use an envvar you set

**Reply by Emilgardis** [reply]
Note also that it goes [by cwd](https://github.com/cross-rs/cross/blob/4090beca3cfffa44371a5bba524de3a578aa46c3/src/file.rs#L218), we might actually want to change that to always be relative to the cargo manifest.

**Reply by NotQuiteApex** [reply]
Is there a way to do this as a passed argument to cross rather than with a config file?

**Reply by Emilgardis** [reply]
no, all config for cross is done via config files, the idea being that you can replace `cross` with `cargo` and it's still compatible.

You can also use env vars, something like `CROSS_BUILD_ENV_VOLUMES="ASSETS=blabla"`, split by whitespace to add new entries
